src/platform_linux/shmem.c

```diff
@@ -497,53 +505,13 @@ platform_shmdestroy(platform_heap_id *hid_out)
    // Reset globals to NULL; to avoid accessing stale handles.
    Heap_id = NULL;
 
-   fraction used_bytes_pct;
-   fraction used_bytes_hwm_pct;
-   fraction free_bytes_pct;
-   used_bytes_pct     = init_fraction(shm_used_bytes, shm_total_bytes);
-   used_bytes_hwm_pct = init_fraction(shm_used_bytes_HWM, shm_total_bytes);
-   free_bytes_pct     = init_fraction(shm_free_bytes, shm_total_bytes);
-
-   // clang-format off
    // Always trace destroy of shared memory segment.
    platform_default_log("Deallocated SplinterDB shared memory "
-                        "segment at %p, shmid=%d."
-                        " Used=%lu bytes (%s, " FRACTION_FMT(4, 2) " %%)"
-                        " Used HWM=%lu bytes (%s, " FRACTION_FMT(4, 2) " %%)"
-                        ", Free=%lu bytes (%s, " FRACTION_FMT( 4, 2) " %%)"
-                        ".\n",
+                        "segment at %p, shmid=%d.\n",
                         shmaddr,
-                        shmid,
-                        shm_used_bytes,
-                        size_str(shm_used_bytes),
-                        (FRACTION_ARGS(used_bytes_pct) * 100),
-                        shm_used_bytes_HWM,
-                        size_str(shm_used_bytes_HWM),
-                        (FRACTION_ARGS(used_bytes_hwm_pct) * 100),
-                        shm_free_bytes,
-                        size_str(shm_free_bytes),
-                        (FRACTION_ARGS(free_bytes_pct) * 100));
-   // clang-format on
+                        shmid);
 
-   fraction freed_last_frag_pct;
-   freed_last_frag_pct = init_fraction(shm_nfrees_last_frag, shm_nfrees);
-
-   // clang-format off
-   platform_default_log("SplinterDB shared memory stats:"
-                        " nfrees=%lu"
-                        ", nfrees-last-small-frag=%lu ("
-                        FRACTION_FMT(4, 2) " %%)"
-                        ", Large fragments in-use HWM=%u (found in-use=%d)"
-                        ", consumed=%lu bytes (%s)"
-                        ".\n",
-                        shm_nfrees,
-                        shm_nfrees_last_frag,
-                        (FRACTION_ARGS(freed_last_frag_pct) * 100),
-                        nlarge_frags_inuse_HWM,
-                        large_frags_found_in_use,
-                        used_by_large_frags_bytes,
-                        size_str(used_by_large_frags_bytes));
```

Comment:
This messy chunk of deleted code is now all handled cleanly inside expanded fn `platform_shm_print_usage_stats()`, on L514 below.